languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -37493,6 +37493,110 @@ though, and has since been slightly modified)-->
+      <rulegroup id="CONFUSION_ORTHOGRAPHIQUE" name="confusion orthographique" default="temp_off">
+          <rule>
+              <pattern>
+                  <token>filigramme</token>
+              </pattern>
+              <message>Une erreur d’orthographe semble être détectée.</message>
+              <suggestion suppress_misspelled="yes"><match no="1" regexp_match="(?iu)mme" regexp_replace="ne"/></suggestion>
+              <example correction="filigrane">Il me l'a fait comprendre en <marker>filigramme</marker>.</example>
+          </rule>
+          <rule>
+              <pattern>
+                  <token regexp="yes">rassénér.*</token>
+              </pattern>
+              <message>Une erreur d’orthographe semble être détectée.</message>
+              <suggestion suppress_misspelled="yes"><match no="1" regexp_match="(?iu)nér" regexp_replace="rén"/></suggestion>
+              <example correction="rasséréné">A cette bonne nouvelle, il s'est soudain <marker>rassénéré</marker>.</example>
+          </rule>
+          <rule>
+              <pattern>
+                  <token regexp="yes">omnibul.*</token>
+              </pattern>
+              <message>Une erreur d’orthographe semble être détectée.</message>
+              <suggestion suppress_misspelled="yes"><match no="1" regexp_match="(?iu)mnibul" regexp_replace="bnubil"/></suggestion>
+              <example correction="obnubiler">Elle va <marker>omnibuler</marker> son frère.</example>
+          </rule>
+          <rule>
+              <pattern>
+                  <token regexp="yes">omnubil.*</token>
+              </pattern>
+              <message>Une erreur d’orthographe semble être détectée.</message>
+              <suggestion suppress_misspelled="yes"><match no="1" regexp_match="(?iu)mnubil" regexp_replace="bnubil"/></suggestion>
+              <example correction="obnubiler">Elle va <marker>omnubiler</marker> sa fille.</example>
+          </rule>
+          <rule>
+              <antipattern>
+                  <token>«</token>
+                  <token>scénette</token>
+                  <token>»</token>
+              </antipattern>
+              <pattern>
+                  <token>scénette</token>
+              </pattern>
+              <message>Une erreur d’orthographe semble être détectée.</message>
+              <suggestion suppress_misspelled="yes"><match no="1" regexp_match="scénette" regexp_replace="saynète"/></suggestion>
+              <example correction="saynète">Cette <marker>scénette</marker> est intéressante.</example>
+          </rule>
+          <rule>
+              <pattern>
+                  <token skip="-1">jours</token>
+                  <marker>
+                      <token>bimestriel</token>
+                  </marker>
+              </pattern>
+              <message>Une erreur d’orthographe semble être détectée.</message>
+              <suggestion><match no="2" regexp_match="bimestriel" regexp_replace="bimensuel"/></suggestion>
```

Comment:
What if someone says "Il y a deux jours, on a eu notre meeting bimestriel" (qui a lieu tous les deux mois)? Can the feminine form be included?

Reply:
The sentence is possible but I think that "bimestriel" is less used than "bimensuel". 
Yes, the feminine form can be included.